hack/install-nm.sh

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash -e
 
-# TODO: Iterate all the nodes
+ssh=kubevirtci/cluster-up/ssh.sh
+kubectl=kubevirtci/cluster-up/kubectl.sh
 
-node01_id=$(docker ps |grep node01 |awk '{print $1}')
+function install_nm_on_node() {
+    node=$1
+    $ssh $node sudo -- yum install -y NetworkManager NetworkManager-ovs
+    $ssh $node sudo -- systemctl daemon-reload
+    $ssh $node sudo -- systemctl restart NetworkManager
+    echo "Check NetworkManager is working fine on node $node"
+    $ssh $node -- nmcli device show > /dev/null
+}
 
-echo 'Install NetworkManager on the node'
-docker exec $node01_id ssh.sh sudo yum install -y NetworkManager
```

Comment:
I am happy we can remove this, there were a kubecirtci cli bug that was not blocking until command end.